Data/models/PlaylistInfo.swift

```diff
@@ -72,6 +73,15 @@ public struct PlaylistInfo: Codable {
         return nil
     }
     
+    private static func fixSchemelessURLs(src: String, pageSrc: String) -> String {
```

Comment:
Maybe expose and write an unit test for it

Reply:
Fixed.